source/tutorials/fmt-cdiff-khanna.rst

```diff
@@ -506,6 +587,58 @@ We're also often interested in performing a differential abundance test at a spe
 .. question::
    Which genera differ in abundance across Subject? In which subject is each genus more abundant?
 
+
+Using q2-sample-classifier to predict sample characteristics
+------------------------------------------------------------
+
+Looks like CDI leads to a state of microbial dysbiosis. Could we use features of that dysbiosis state to diagnose CDI? Let's use :doc:`q2-sample-classifier <sample-classifier>` to find out. We will train a Random Forest classifier on a subset of our samples, and test the predictive accuracy of this model using another subset of our samples to determine how well this model generalizes to unseen samples. We will train our model to distinguish between pre- and post-FMT samples as a proxy measurement of current CDI:
+
+.. command-block::
+
+  qiime sample-classifier classify-samples \
+    --i-table disease-table.qza \
+    --m-metadata-file sample_metadata.tsv \
+    --m-metadata-column day_since_fmt \
+    --p-n-estimators 200 \
+    --p-random-state 666 \
+    --output-dir sample-classifier
+
+
+This action outputs a bunch of artifacts and visualizations, but we will focus on ``accuracy_results.qzv`` for now; this visualization shows us how frequently our trained classifier predicted CDI (or more precisely, how well it could distinguish sample pre- and post-FMT). We see an overall accuracy rate of 86.6%, much better than the error rate achieved by random guessing (53.3%). So this method is useful for differentiating pre- and post-FMT samples (and identifying differential features, as we will test below), but probably still not good enough to be considered a useful diagnostic tool!

```

Comment:
Is it worth at least mentioning the other files?

Reply:
no, these are mostly intermediate artifacts that are provided for convenience and would just clog up this tutorial to discuss here; I have added a link to the sample classifier tutorial indicating that more information on these outputs can be found there.